src/main/java/seedu/address/logic/parser/CliSyntax.java

```diff
@@ -17,5 +17,5 @@
     public static final Prefix PREFIX_AND = new Prefix("/AND");
     public static final Prefix PREFIX_OR = new Prefix("/OR");
     public static final Prefix PREFIX_NOT = new Prefix("/NOT");
-    public static final Prefix PREFIX_COE_EXPIRY = new Prefix("coe/");
+    public static final Prefix PREFIX_COE_EXPIRY = new Prefix("ex/");
```

Comment:
Is this the solution? Could you explain how this solution works?

Reply:
Ig we already discussed!